README.md

```diff
@@ -81,17 +78,9 @@ Once the PR is reviewed and merged, the changes will appear on Graph Explorer in
        `"skipTest": `<br>
 1. The `humanName` value should be in **small caps** only.
 1. The `requestUrl` value should be a relative url, starting from the version, i.e. `"requestUrl": "/v1.0/me/onenote/notebooks/{notebook-id}/sections"` leave out the `https://graph.microsoft.com` part of the url.
-### Using Command Line or PowerShell:
-
-### 1. Clone the repo (If you have write access)
-Open the command line terminal and paste this command to clone or fork this repo: <br/>
-`git clone https://github.com/microsoftgraph/microsoft-graph-devx-content.git microsoft-graph-devx-content
-cd microsoft-graph-devx-content/sample-queries
-`
-
-**Please Note: If you don't have write access or you are a new contributor, fork your copy of the repo to your GitHub.**
```

Comment:
You can also update this:
**Please Note: If you don't have write access or you are a new contributor, fork your own copy of this repo by clicking on the **Fork** tab at the top right of this page.**